server/src/test/java/org/spine3/server/entity/Given.java

```diff
@@ -73,22 +71,27 @@ static CreateProject createProject() {
         private boolean isValidateMethodCalled = false;
 
         static TestEntity newInstance(String id) {
-            final TestEntity entity = new TestEntity(id);
-            return entity;
+            final TestEntity result = org.spine3.test.Given.entityOfClass(TestEntity.class)
+                                        .withId(id)
+                                        .build();
+            return result;
         }
 
         static TestEntity withState() {
-            final Project state = newProject();
-            final int version = 3;
-            final Timestamp whenModified = getCurrentTime();
-            final TestEntity entity = new TestEntity(newUuid());
-            entity.setState(state, version, whenModified);
-            return entity;
+            final TestEntity result = org.spine3.test.Given.entityOfClass(TestEntity.class)
```

Comment:
Can we avoid FQN name here?

I would like to avoid multiple `Given` classes in tests. If we need to extend the API for the main `Given` class, let's discuss.